examples/globe_wfs_extruded.html

```diff
@@ -161,6 +161,7 @@
                 update: itowns.FeatureProcessing.update,
                 convert: itowns.Feature2Mesh.convert({
                     color: colorBuildings,
+                    // batchId: (prop, id) => id,
```

Comment:
it's not necessary, this is in documentation

Reply:
Not necessary, but helpful. I agree that it would be better to have an example that actually uses the feature...

Reply:
Yes, example is better, moreover it avoids a regression, but not as much as a test.
I'm not for commented code but for documentation with example.

Reply:
@mbredif, it just remains to add examples in the documentation.
```js
/**
* @example <caption>Example usage of batchId with featureId.</caption>
* view.addLayer({
*     id: 'WFS Buildings',
*     type: 'geometry',
*     update: itowns.FeatureProcessing.update,
*     convert: itowns.Feature2Mesh.convert({
*         color: colorBuildings,
*         batchId: (property, featureId) => featureId,
*         altitude: altitudeBuildings,
*         extrude: extrudeBuildings }),
*     onMeshCreated: function scaleZ(mesh) {
*         mesh.scale.z = 0.01;
*         meshes.push(mesh);
*     },
*     filter: acceptFeature,
*     source,
* });
*
* @example <caption>Example usage of batchId with property.</caption>
* view.addLayer({
*     id: 'WFS Buildings',
*     type: 'geometry',
*     update: itowns.FeatureProcessing.update,
*     convert: itowns.Feature2Mesh.convert({
*         color: colorBuildings,
*         batchId: (property, featureId) => property.house ? 10 : featureId,
*         altitude: altitudeBuildings,
*         extrude: extrudeBuildings }),
*     onMeshCreated: function scaleZ(mesh) {
*         mesh.scale.z = 0.01;
*         meshes.push(mesh);
*     },
*     filter: acceptFeature,
*     source,
* });
**/
```